a4/a4.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> ShopQuick </title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <style>
+      article {
+        margin: 250px;
+        font-size: 40px;
+        text-align: center;
+      }
+      #border {
+        border-style: solid;
+        border-width: 5px;
+      }
+      #space {
+        margin: 30px auto;
+      }
+    </style>
+  </head>
+  <body>
+
+    <div class="jumbotron text-center">
+      <h1> Welcome to ShopQuick</h1>
+    </div>
+
+    <div class="container">
+      <div class="row" id="border">
+        <div class="col-sm-4">
+          <h3>To Eat</h3>
+        </div>
+        <div class="col-sm-4">
+          <h3>For Work</h3>
+        </div>
+        <div class="col-sm-4">
+          <h3>For Fun</h3>
+        </div>
+      </div>
+    </div>
+
+    <div class="container" id="space">
+      <div class="row">
+        <div class="col-sm-4">
+          <h3></h3>
+        </div>
+        <div class="col-sm-4">
+          <h3>Newly added items!</h3>
+        </div>
+        <div class="col-sm-4">
+          <h3></h3>
+        </div>
+      </div>
+    </div>
+
+    <div class="container">
```

Comment:
Look around the Bootstrap site for examples of what you want to do (or similar).

I found this after clicking through the components one after another: https://getbootstrap.com/docs/3.3/components/#thumbnails

Bootstrap 4 elevated this pattern to a named thing in the system: https://getbootstrap.com/docs/4.3/components/card/